src/yaourt.sh.in

```diff
@@ -127,6 +128,15 @@ yaourt_install_packages() {
 	prepare_status_list
 	SP_ARG="" sync_packages "${ARGS[@]}" || return $?
 	analyse_status_list
+
+	# Trigger Backup
+	rnum=${#ARGS[@]}
+	fnum=${#ERROR_PKGS[@]}
+	if [[ "$((rnum - fnum))" -gt "0" ]]; then
+		load_lib gist_backup
+		gist_backup_poll
```

Comment:
Not true- the user is never prompted to install gist unless they run --gist-backup on their own.